src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -1,6 +1,7 @@
 import { apiRequest } from 'platform/utilities/api';
 import { refreshProfile } from 'platform/user/profile/actions';
 import recordEvent from 'platform/monitoring/record-event';
+import { inferAddressType } from '../../../../../applications/letters/utils/helpers';
```

Comment:
this can just be `import { inferAddressType } from 'applications/letters/utils/helpers';`

Reply:
Fixed